File: week1/MarshalLee/ContainsDuplicate.cs
public class Solution
{
    public bool ContainsDuplicate(int[] nums)
    {
        foreach (int number in nums)
        {
            if (Array.LastIndexOf(nums, number) != Array.IndexOf(nums, number))
            {
                return true;
            }
        }
        return false;
    }
}

File: week1/Erenimo3442/contains_duplicate.py
class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
        count = dict()
        for number in nums:
            count[f"{number}"] = count.get(f"{number}", 0) + 1
        for value in count.values():
            if value > 1:
                return True
                break
        return False